### PR TITLE
Rename v4 subtoc

### PR DESCRIPTION
**Pull request by suehighmoor**
changed to OpenShift v4 for consistency with v3 subtoc

### CODE REVIEW

File: articles/openshift/oshift-ref-deploy-app-into-service-mesh.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ reviewer:
 lastreviewed: 22/07/2020
 
 toc_rootlink: Reference
-toc_sub1: v4
+toc_sub1: OpenShift v4
```
Comment (andrew-garner) [inline articles/openshift/oshift-ref-deploy-app-into-service-mesh.md:10]: Sorry I missed this one. Could we make it "OpenShift v4.x" please. That's the format we ended up deciding on for OpenShift v3.x docs.

**Review by andrew-garner — CHANGES_REQUESTED** [review]

File: articles/openshift/oshift-ref-deploy-app-into-service-mesh.md (hunk 1 above)
Comment (suehighmoor) [reply articles/openshift/oshift-ref-deploy-app-into-service-mesh.md]: Done!

**Review by andrew-garner — APPROVED** [review]
LGTM, thank you :)

**Review by koneill94 — APPROVED** [review]
LGTM

**Review by stevemul — APPROVED** [review]
LGTM

**Pull request merged**